ci/playbooks/multinode-customizations.yml

```diff
@@ -129,12 +129,12 @@
 
     - name: Generate an ssh keypair
       ansible.builtin.command:
-        cmd: ssh-keygen -t ed25519 -f ~/.ssh/id_ed25519 -P ''
```

Comment:
why switch to ecdsa? ed25519 is usually flagged as better. Is it due to FIPS?
Otherwise, OK for a generic name, though it will prevent any "default behavior" in ssh.

Reply:
Exactly, ed25519 doesn't work with FIPS, and ecdsa is what nova is using.